library/tasks/Attach10xBarcodes.wdl

```diff
@@ -1,32 +1,57 @@
 task Attach10xBarcodes {
-  # attaches barcodes found in r1 and i1
+  File r1_fastq
```

Comment:
personal preference: lets group the fastq parameters together (move the unmapped_bam) down one line. Also lets retain the r2 -> `r2_unmapped_bam`.